src/dev/i18n/utils.js

```diff
@@ -84,3 +94,71 @@ export function* traverseNodes(nodes) {
     }
   }
 }
+
+/**
+ * @param {string[]} valuesKeys array of "values" property keys
+ * @param {string} defaultMessage "defaultMessage" value
+ * @throws if "values" and "defaultMessage" don't correspond to each other
+ */
+export function checkValuesProperty(valuesKeys, defaultMessage, messageId) {
+  const defaultMessageReferences = defaultMessage.match(VALUES_REFERENCES_REGEX);
+
+  if (!defaultMessageReferences) {
+    return;
+  }
+
+  const defaultMessageReferencesKeys = defaultMessageReferences.map(
+    message => message.match(EXTRACT_VALUE_KEY_FROM_REFERENCE_REGEX)[0]
```

Comment:
`VALUES_REFERENCES_REGEX` matches a substring `{value }`, but here `EXTRACT_VALUE_KEY_FROM_REFERENCE_REGEX` can not match `value` from this substring because of space placed before closing curly brace although it should.  I think this two RegExps should be aligned so that `EXTRACT_VALUE_KEY_FROM_REFERENCE_REGEX` can extract values from all strings matched by `VALUES_REFERENCES_REGEX`. What do you think about it?